src/nwbinspector/testing/_testing.py

```diff
@@ -44,26 +47,28 @@ def check_streaming_tests_enabled() -> tuple[bool, Optional[str]]:
     if not have_remfile:
         failure_reason += "The `remfile` package is not installed on the system."
 
-    failure_reason = None if failure_reason == "" else failure_reason
-    return streaming_enabled and not environment_skip_flag_bool and have_dandi, failure_reason
+    return_failure_reason: Optional[str] = None
+    if failure_reason != "":
+        return_failure_reason = failure_reason
+
+    return streaming_enabled and not environment_skip_flag_bool and have_dandi, return_failure_reason
 
 
-def generate_testing_files():  # pragma: no cover
+def generate_testing_files() -> None:  # pragma: no cover
     """Generate a local copy of the NWB files required for all tests."""
     generate_image_series_testing_files()
 
 
-def generate_image_series_testing_files():  # pragma: no cover
+def generate_image_series_testing_files() -> None:  # pragma: no cover
     """Generate a local copy of the NWB files required for the image series tests."""
     assert get_package_version(name="pynwb") == Version("2.1.0"), "Generating the testing files requires PyNWB v2.1.0!"
+    assert TESTING_FILES_FOLDER_PATH is not None, "The `TESTING_FILES_FOLDER_PATH` environment variable is not set!"
 
-    testing_config = load_testing_config()
-
-    local_path = Path(testing_config["LOCAL_PATH"])
-    local_path.mkdir(exist_ok=True, parents=True)
+    testing_folder = Path(TESTING_FILES_FOLDER_PATH)
+    testing_folder.mkdir(exist_ok=True)
```

Comment:
```suggestion
    testing_folder.mkdir(exist_ok=True, parents=True)
```

might still be needed if nested directories?

Reply:
I caution against doing that very stringly, because if a target path is formatted incorrectly by mistake (say, like a WindowsPath) then calling `parents=True` will cause that folder to be created even if the root makes no sense for the OS (such as UNIX)

Ran into that one a couple times before.

If you want to ensure specific parents exist, I suggest going through them one at a time to make sure they exist (from top to bottom of course)